src/sandstorm/web-session.capnp

```diff
@@ -463,7 +463,7 @@ interface WebSession @0xa50711a14d35a8ce extends(Grain.UiSession) {
   }
 
   interface WebSocketStream {
-    sendBytes @0 (message :Data);
+    sendBytes @0 (message :Data) -> stream;
```

Comment:
Hmm, I wonder if this is going to be problematic since there's no `end()` or `done()` method. That goes against the recommended pattern and I haven't thought a whole lot about what exactly happens in this case.

I _think_ that if you queue up a bunch of streaming calls and then drop the capability, all calls will be delivered before the destination object is destroyed...

You might not find out if an exception was thrown on the server side, but I guess that's not expected anyway for WebSocket... the application has to detect problems by replying.

So......... it might work?